web-ng/src/app/services/dialog/dialog.service.spec.ts

```diff
@@ -17,17 +17,49 @@
 import { TestBed } from '@angular/core/testing';
 
 import { DialogService } from './dialog.service';
-import { MatDialogModule } from '@angular/material/dialog';
+import { MatDialog, MatDialogModule } from '@angular/material/dialog';
+import { ConfirmationDialogComponent } from '../../components/confirmation-dialog/confirmation-dialog.component';
 
 describe('DialogService', () => {
   let service: DialogService;
+  let dialogSpy: jasmine.Spy;
 
   beforeEach(() => {
     TestBed.configureTestingModule({ imports: [MatDialogModule] });
     service = TestBed.inject(DialogService);
+
+    dialogSpy = spyOn(TestBed.inject(MatDialog), 'open');
```

Comment:
Do we need to spy on a real `MatDialog`, or can we just used `createSpyObj` to create a dialog with the methods we need to spy on"?

Reply:
Hi @os-micmec Gentle ping!

Reply:
Done!